members/lisa_je.md

```diff
@@ -0,0 +1,34 @@
+---
+
+layout: default
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/lisa_je.jpg" alt="Placeholder Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Lisa Je (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab** National Science Foundation Graduate Research Fellow (NSF GRFP)
```

Comment:
I would put NSF GRFP under cool stuff I've done, while I think you should probably reword your Role in the Cersonsky Lab. I think it's OK to elaborate your role as a senior mentor in 1-2 sentences, and if you want, also link to your official Zavanlehn webpages.

Reply:
I would agree putting you under "Collaborators and Visitors" -- I don't want to cause any confusion by publicly appearing to claim you as my own officially :)